packages/eslint-config-server/index.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
 
   rules: {
     'prettier/prettier': 'warn',
+    'no-unused-var': 0
```

Comment:
Ahhh yep I also had this in place before.. whats funny about it, is that lint-staged has no problem with that. So runnung:
```bash
npx lint-staged
```
will work perfectly fine on all apps. Mainly implemented the whole thing for runnung pre-commit linting so I did not test it for the standard eslint command again.
Not super happy with it but I guess its good enough for now.

BUT I think it should rather be 
```js
{
  ...
  'no-unused-vars': 0, // with an 's' a the end
}
```